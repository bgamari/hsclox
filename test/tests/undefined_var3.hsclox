var c = 1;
var a = 1;
b = "hi";
