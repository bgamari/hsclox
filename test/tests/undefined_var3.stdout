Undefined variable 'b'.
[line 3] in script
